x-pack/plugins/ml/public/cases/index.tsx

```diff
@@ -0,0 +1,111 @@
+    <EuiButtonIcon
+      data-test-subj="test-attachment-action"
+      onClick={() => {}}
+      iconType="boxesHorizontal"
+      aria-label="See attachment"
+    />
+  );
+};
+
+export function registerCasesAttachment(
+  cases: CasesUiSetup,
+  coreStart: CoreStart,
+  pluginStart: MlStartDependencies
+) {
+  cases.attachmentFramework.registerPersistableState({
+    id: ANOMALY_SWIMLANE_EMBEDDABLE_TYPE,
+    icon: PLUGIN_ICON,
+    // TODO check where this name is presented
+    displayName: i18n.translate('xpack.ml.cases.anomalySwimLane.displayName', {
+      defaultMessage: 'Anomaly swim lane',
+    }),
+    getAttachmentViewObject: () => ({
+      event: (
+        <FormattedMessage
+          id="xpack.ml.cases.anomalySwimLane.embeddableAddedEvent"
+          defaultMessage="added the Anomaly Swim Lane embeddable"
+        />
+      ),
+      timelineAvatar: PLUGIN_ICON,
+      actions: <AttachmentActions />,
+      children: React.lazy(() => {
+        return Promise.resolve().then(() => {
+          const EmbeddableComponent = getAnomalySwimLaneEmbeddableComponent(coreStart, pluginStart);
+
+          return {
+            default: React.memo((props: PersistableStateAttachmentViewProps) => {
+              const { persistableStateAttachmentState } = props;
+
+              const inputProps =
+                persistableStateAttachmentState as unknown as AnomalySwimlaneEmbeddableInput;
+
+              return (
+                <>
+                  <EuiDescriptionList
+                    compressed
+                    type={'inline'}
+                    listItems={[
+                      {
+                        title: (
+                          <FormattedMessage
+                            id="xpack.ml.cases.anomalySwimLane.description.jobIdsLabel"
+                            defaultMessage="Job IDs"
+                          />
+                        ),
+                        description: inputProps.jobIds.join(', '),
+                      },
+                      ...(inputProps.viewBy
+                        ? [
+                            {
+                              title: (
+                                <FormattedMessage
+                                  id="xpack.ml.cases.anomalySwimLane.description.viewByLabel"
+                                  defaultMessage="View by"
+                                />
+                              ),
+                              description: inputProps.viewBy,
+                            },
+                          ]
+                        : []),
+                      {
+                        title: (
+                          <FormattedMessage
+                            id="xpack.ml.cases.anomalySwimLane.description.timeRangeLabel"
+                            defaultMessage="Time range"
+                          />
+                        ),
+                        description: `${inputProps.timeRange.from} - ${inputProps.timeRange.to}`,
```

Comment:
Can the formatting be improved here? I don't think we need `ms` precision:

<img width="545" alt="image" src="https://user-images.githubusercontent.com/7405507/185904134-5bb078fb-f1bb-4e7f-80ee-ec1f61227976.png">

Maybe use the same formatting as used in the Anomaly Explorer here:

<img width="429" alt="image" src="https://user-images.githubusercontent.com/7405507/185904232-76b4a71c-060a-45fb-a01d-9bca9d56a95a.png">

Reply:
added a formatting with the `fieldFormats` plugin here https://github.com/elastic/kibana/pull/138994/commits/78a2738936dc45fbdff477d632fbccfa122adbe8